templates/php.ini.j2

```diff
@@ -64,6 +64,11 @@ auto_append_file =
 
 default_mimetype = "text/html"
 
+{% if php_always_populate_raw_post_data is defined %}
+always_populate_raw_post_data = {{ php_always_populate_raw_post_data }}
```

Comment:
This seems to have been put in as well but is not documented anywhere and is not in the defaults.

Reply:
https://www.php.net/manual/en/ini.core.php - removed in PHP 7.0.0? Maybe remove, here?